tests/php/ORM/MarkedSetTest.php

```diff
@@ -18,6 +18,8 @@ class MarkedSetTest extends SapphireTest
 
     protected static $extra_dataobjects = [
         HierarchyTest\TestObject::class,
+        HierarchyTest\HierarchyOnSubclassTestObject::class,
+        HierarchyTest\HierarchyOnSubclassTestSubObject::class,
```

Comment:
These needed to be added or else we'd now start getting the following when running MarkedSetTest on local

```
SilverStripe\ORM\Connect\DatabaseException: Couldn't run query:

INSERT INTO "HierarchyOnSubclassTest_Object"
 ("ClassName", "Title", "LastEdited", "Created")
 VALUES
 (?, ?, ?, ?)

Table 'ss_tmpdb_1746582023_8776611.HierarchyOnSubclassTest_Object' doesn't exist
```